babel.config.js

```diff
@@ -1,6 +1,19 @@
 module.exports = {
   presets: [
-    ['@babel/env', { exclude: ['transform-regenerator', 'transform-async-to-generator'] }],
+    [
+      '@babel/env',
+      {
+        targets: {
+          node: 10,
+          browsers: [
+            'last 2 chrome versions',
+            'last 2 firefox versions',
+            'last 2 safari versions',
+            'last 2 edge versions',
+          ],
```

Comment:
I want to call out this change in case anyone has any concerns. From #214 it seems like we're okay with not supporting IE 11.

The reason I did this isn't really for smaller bundles but that's a nice benefit, I did this is because there's a bug in Babel which means Babel compiled classes can't extend native classes. (https://github.com/babel/babel/issues/9367)